README.md

```
@@ -1,58 +1,113 @@
-# or start the server and open the app in a new browser tab
-npm run dev -- --open
-```
-
-Everything inside `src/lib` is part of your library, everything inside `src/routes` can be used as a showcase or preview app.
-
-## Building
+### 2. Setup Provider
+Add the ThirdwebSvelteProvider to your `src/routes/layout.svelte`:
+```svelte
+<script>
+	import { ThirdwebSvelteProvider } from '@holdex/thirdweb-svelte';
```

Markdownlint (0.37.0)

Column: 1
Hard tabs

(MD010, no-hard-tabs)